DataFormats/simulation/include/SimulationDataFormat/MCTruthContainer.h

```diff
@@ -115,6 +115,14 @@ class MCTruthContainer
     mTruthArray.emplace_back(element);
   }
 
+  // convenience interface to add multiple labels at once
+  void addElements(uint dataindex, gsl::span<const TruthElement> elements)
+  {
+    for (auto& e : elements) {
+      addElement(dataindex, e);
+    }
+  }
```

Comment:
@davidrohr : Is this what you need?

Potentially we could make this a bit more generic by allowing
```
template<typename OtherElement>
void addElements(uint dataindex, gsl::span<const OtherElement> elements)
{
   for (auto& e : elements) {
       addElement(dataindex, e);
   }
}
```
which is something that we needed in the TOF (when for instance OtherElement is a daughter of TruthElement but we only want to store the base information). Could add a concept/static_assert to constrain this a bit.